представления/вован-зт/выскакивать/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>popup</title>
+</head>
+<body>
+   
+    <nav class="navigation">
+        <ul class="navigation__list">
+            <li class="navigation__item">
+                <a href="#"><img class="navigation__image" src="icons/bell.png" alt="bell1"> </a>
+            </li>
+            <li class="navigation__item">
+                <a href="#"><img class="navigation__image" src="icons/bell.png" alt="bell2"></a>
+            </li>
+            <li class="navigation__item navigation__item-modal">
+                <input  class="navigation__check_input" type="checkbox" name="modal" id="modal">
+                <label  for="modal" ><img class="navigation__image" src="icons/popup-button.png" alt="popup"></label> 
+
+                <div class="modal">
+
+                    <input  class="navigation__check_input" type="checkbox" name="modal2" id="modal2">
```

Comment:
When the modal is open focus must be on the first interactive element in this modal, usually, it's close button. but in our case when the user opens the modal and presses the` Tab` key focus must go to the first link in modal)

Reply:
Still not fixed :cry: